static/templates/settings/bot-settings.handlebars

```diff
@@ -84,11 +84,11 @@
     <div id="get_api_key_box" class="side-padded-container">
       {{#tr this}}
       <p>Please re-enter your password to confirm your identity.
-        (<a href="/accounts/password/reset/" target="_blank">Forgotten it?</a>)</p>
+        <a href="/accounts/password/reset/" target="_blank">{{t "Forgotten it?" }}</a></p>
       {{/tr}}
       <form action="/json/fetch_api_key" method="post" class="form-horizontal">
         <div class="control-group">
-          <label for="password" class="control-label">{{t "Current password" }}</label>
+          <label for="password" class="control-label">{{t "Current password" }}</label>&nbsp;
```

Comment:
Understood! Thanks a lot for the explanation.

I'll see if I can do some modifications to our development docs to make this clearer for other contributors :)